src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp

```diff
@@ -1263,6 +1867,30 @@ INSTANTIATE_TEST_SUITE_P(smoke_JIT_AVX512_DW_GroupConv, GroupConvolutionLayerCPU
 /* ============= JIT AVX5122 PLANAR Convolution (not supported with groups) ============= */
 /* ============================================= */
 
+/* ============= expect failed brgemm GroupConvolution  ============= */
+const std::vector<CPUSpecificParams> CPUParams_Failed_Brgemm_2D = {
+        conv_avx512_2D_nspc_brgconv,
+        conv_avx512_2D_nspc_brgconv_amx
+};
```

Comment:
So here we have brgconv_avx512_amx in priority list and expect also brgconv_avx512_amx as selected primitive type. Why do we call it a "failed" brgconv then? I thought that in the failed case brgconv primitive will not be chosen and avx512 will be use instead. Have I missed something?

Reply:
already replace "failed" with "fallback", please help to review

Reply:
Discussed, the problem is that for this test case we expect primitive type to not match